server/models/Reviews.js

```diff
@@ -0,0 +1,31 @@
+const { Schema, model } = require('mongoose');
+
+const ReviewsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  reviews: [
```

Comment:
What is the purpose of having the field reviews as an array? Because based on how reviews are being created currently, each new Reviews document has a an array of reviews within it (but they only contain one review) ??